Gemfile

```
@@ -80,3 +80,6 @@ gem 'kaminari'
 
 # N+1検出用
 gem 'bullet'
+
+# タグ生成用
+gem 'acts-as-taggable-on', '~>6.0'
```

6.5があるのでそちらを使うようにしましょうか
```suggestion
gem 'acts-as-taggable-on', '~>6.5'
```
https://github.com/mbleigh/acts-as-taggable-on/tags